Doc/library/xmlrpc.client.rst

```diff
@@ -59,9 +56,17 @@ between conformable Python objects and XML on the wire.
    presented as :class:`bytes` objects; this flag is false by default.
    :class:`datetime.datetime`, :class:`bytes` and :class:`bytearray` objects
    may be passed to calls.
+   The *headers* list can be used to set HTTP headers to each request. It is
+   composed of tuple of header name and value.
```

Comment:
If I understand correctly, this is supposed to be a `list` of `(name, value)` pairs? If so, I would reword it something more like this:

```rst
The *headers* parameter is an optional list of HTTP headers to send
with each request, expressed as a :class:`list` of :class:`tuple`s representing
the header name and value. (e.g. `[('HeaderName', 'value')]`).
```